build.sbt

```diff
@@ -66,9 +65,9 @@ lazy val addJavaOptions = javaOptions ++= {
   )
 }
 
-addCommandAlias("hot", "compilation/jmh:run HotScalacBenchmark")
+addCommandAlias("hot", "compilation/jmh:runMain scala.bench.ScalacBenchmarkRunner HotScalacBenchmark")
 
-addCommandAlias("cold", "compilation/jmh:run ColdScalacBenchmark")
+addCommandAlias("cold", "compilation/jmh:runMain scala.bench.ScalacBenchmarkRunner ColdScalacBenchmark")
```

Comment:
We could set `mainClass` to `scala.bench.ScalacBenchmarkRunner `, as demonstrated in https://github.com/ktoso/sbt-jmh/blob/master/plugin/src/sbt-test/sbt-jmh/custom-runner/build.sbt to make this runner the default.